basic-principles.md

```diff
@@ -0,0 +1,100 @@
+---
+title: SQL 开发规范及基本原则
+category: principle
+---
+
+# SQL 开发规范及基本原则
+
+## 对象命名规范
+
+* 命名建议使用具有意义的英文词汇，词汇中间以下划线分隔；
+* 命名只能使用英文字母、数字、下划线；
+* 避免用 TiDB 的保留字如：group，order 等作为单个字段名。
+* 建议所有数据库对象使用小写字母。
+
+## 建表删表规范
+
+基本原则：表的建立在遵循表命名规范前提下，建议业务应用内部封装建表删表语句增加判断逻辑，防止业务流程异常中断。例如：create table if not exists table_name 或者 drop table if exists table_name 语句建议增加 if 判断，避免应用侧由于表的改动造成的异常中断。		
```

Comment:
Please add inline code format for code, variable, parameters, etc. for this PR.

Reply:
added